.github/workflows/remark-lint.yml

```
@@ -0,0 +1,21 @@
+name: remark lint
+on: [push]
+jobs:
+  run:
+    name: run
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: set node.js 14.x
+      uses: actions/setup-node@master
+      with:
+        version: 14.x
```

```suggestion
        node-version: 14.x
```
I saw this from https://github.com/Financial-Times/o-assets/actions/runs/122695252
>Input 'version' has been deprecated with message: The version property will not be supported after October 1, 2019. Use node-version instead

---

thank you:)

i've abandoned this for a while, and filed an issue on the preset repo: https://github.com/Financial-Times/remark-preset-lint-origami-component/issues/5

it was a rabbit hole, would rather continue the good work and look at it another time